core/utils/idgenerator.js

```diff
@@ -29,5 +29,5 @@ Blockly.utils.IdGenerator.nextId_ = 0;
  * @return {string} The next unique identifier.
  */
 Blockly.utils.IdGenerator.getNextUniqueId = function() {
-  return 'blockly:' + (Blockly.utils.IdGenerator.nextId_++).toString(36);
+  return 'blockly-' + (Blockly.utils.IdGenerator.nextId_++).toString(36);
```

Comment:
What is the reasoning behind this change?

Reply:
The comment above says: "Use only ASCII letters, digits, '_', '-' and '.'".
In addition, css class names can not have `:` in the name.

Reply:
Ok, I think we should add the api change label for this then. From my understanding since we don't have a way to add classes to the toolbox categories, people sometimes rely on some hack around using the category ids. It's not really supported but I think would still be a nice heads up.

Reply:
Done.